stories/pdf-viewer/pdf-viewer.stories.js

```diff
@@ -0,0 +1,198 @@
+
+##Usage
+**Use the PDF viewer if:**
+
+- You want your app to display PDF files on all devices and platforms.
+- You want the users of your app to be able to preview their documents as PDF files right beside your app.
+- You need to ensure the consistent behavior of PDF files across your app.
+
+**Do not use the PDF viewer if:**
+
+- You need to provide an interactive PDF file (such as a data input form).
+
+Ensure that the height of the dialog can fit the height of the iframe without scrolling the dialog itself; there should not be two scroll bars within the dialog.
+
+`,
+        tags: ['f3', 'a11y']
+    }
+};
+
+export const dialog = () => `
+<div class="fd-dialog-docs-static fd-pdf-viewer fd-dialog fd-dialog--active">
+    <div class="fd-pdf-viewer__dialog fd-dialog__content fd-dialog__content--l" style="min-height: 28rem;" role="dialog" aria-modal="true" aria-labelledby="dialog-title-1">
+        <header class="fd-dialog__header fd-bar fd-bar--header">
+            <div class="fd-bar__left">
+                <div class="fd-bar__element">
+                    <h2 class="fd-title fd-title--h5" id="dialog-title-1">
+                        PDF Viewer title
+                    </h2>
+                </div>
+            </div>
+        </header>
+        <div class="fd-dialog__body">
+            <iframe
+              class="fd-pdf-viewer__content"
+              title='PDF viewer'
+              style="min-height: 20rem;"
+              src="https://sapui5.hana.ondemand.com/test-resources/sap/m/demokit/sample/PDFViewerEmbedded/sample.pdf#view=FitH"}
+              frameBorder='0'></iframe>
+        </div>
+        <footer class="fd-dialog__footer fd-bar fd-bar--footer">
+            <div class="fd-bar__right">
+                <div class="fd-bar__element">
+                    <button class="fd-dialog__decisive-button fd-button fd-button--compact">
+                        Download
+                    </button>
+                </div>
+                <div class="fd-bar__element">
+                    <button class="fd-dialog__decisive-button fd-button fd-button--compact">
+                        Close
+                    </button>
+                </div>
+            </div>
+        </footer>
+    </div>
+</div>
+`;
+
+dialog.storyName = 'Dialog';
+dialog.parameters = {
+    docs: {
+        iframeHeight: 500,
+        storyDescription: 'When displayed in a dialog, download and close buttons are in the footer. The PDF Viewer dialog does not support resizing. For mobile, ensure the modal takes up the entire screen. Refer to the documentation for Dialog.'
+    }
+};
+
+export const mobileDialog = () => `
+<div class="fd-dialog-docs-static fd-pdf-viewer fd-dialog fd-dialog--active">
+    <div class="fd-pdf-viewer__dialog fd-dialog__content fd-dialog__content--mobile" style="min-height: 28rem;" role="dialog" aria-modal="true" aria-labelledby="dialog-title-1">
```

Comment:
This is inconsistent naming. Dialog follows structure:
```
- Dialog
    - Dialog Content
       - Dialog Content Elements
```
and PDF Viewer which is build as a composition/extension of the Dialog i using `__dialog` on `__content` level. Please change the naming to avoid confusion.